logger/options_test.go

```diff
@@ -12,26 +12,26 @@ func TestOptions_Validate(t *testing.T) {
 	testCases := []struct {
 		label   string
 		options Options
-		valid   bool
+		errStr  string
 	}{
-		{"Base case", Options{}, true},
-		{"With options", Options{Level: "error", Hostname: "foo", App: "app", IPAddress: "127.0.0.1", MacAddress: "C0:FF:EE:C0:FF:EE"}, true},
-		{"App length", Options{App: strings.Repeat("a", 33)}, false},
-		{"Env length", Options{Env: strings.Repeat("a", 33)}, false},
-		{"Hostname length", Options{Hostname: strings.Repeat("a", 33)}, false},
-		{"Level length", Options{Level: strings.Repeat("a", 33)}, false},
-		{"Invalid MacAddress", Options{MacAddress: "in:va:lid"}, false},
-		{"Invalid Hostname", Options{Hostname: "-"}, false},
-		{"Invalid IPAddress", Options{IPAddress: "localhost"}, false},
+		{"Base case", Options{}, ""},
+		{"With options", Options{Level: "error", Hostname: "foo", App: "app", IPAddress: "127.0.0.1", MacAddress: "C0:FF:EE:C0:FF:EE"}, ""},
+		{"App length", Options{App: strings.Repeat("a", 83)}, "One or more invalid options:\nApp: length must be less than 80\n"},
+		{"Env length", Options{Env: strings.Repeat("a", 83)}, "One or more invalid options:\nEnv: length must be less than 80\n"},
+		{"Hostname length", Options{Hostname: strings.Repeat("a", 83)}, "One or more invalid options:\nHostname: length must be less than 80\n"},
+		{"Level length", Options{Level: strings.Repeat("a", 83)}, "One or more invalid options:\nLevel: length must be less than 80\n"},
+		{"Invalid MacAddress", Options{MacAddress: "in:va:lid"}, "One or more invalid options:\nMacAddress: Invalid format\n"},
+		{"Invalid Hostname", Options{Hostname: "-"}, "One or more invalid options:\nHostname: Invalid format\n"},
+		{"Invalid IPAddress", Options{IPAddress: "localhost"}, "One or more invalid options:\nIPAddress: Invalid format\n"},
```

Comment:
There's two features that aren't currently being tested:
1. Multiple incorrect parameters
2. A populated `issues` array